### DISCUSSION TITLE
能否通过浏览器插件实现网站浏览时间统计

### DISCUSSION THREAD

**Discussion by XiaoDuComrade** [Planshit/Tai#181] [Ideas]
开发者您好！我了解到有一款名叫《网费很贵》的chrome浏览器插件实现了对于网站浏览时间的详细统计功能。同时我也了解到有一些软件通过浏览器插件实现信息通信，比如素材收集类软件billfish。能否通过浏览器插件更方便，准确，高效，节约系统资源的实现网站浏览时间统计呢？仅仅是一个想法，我并不是软件开发者，不知道可行性以及效果如何。

**Comment by XiaoDuComrade** [comment]
![image](https://github.com/Planshit/Tai/assets/69492720/0b0ca4e0-0537-4d65-ae07-4747506679f4)
https://github.com/sheepzh/timer/tree/main

**Comment by Wenbobobo** [comment]
今天刚安装，发现打开记录网站时间后Edge 100%崩溃，关闭后正常，期间占用也不高。也推荐用插件或者继续完善下功能吧
总之好看又实用，先给star了！ヾ(•ω•`)o
